src/otx/core/model/entity/visual_prompting.py

```diff
@@ -30,46 +49,918 @@ class OTXVisualPromptingModel(
+        self._parameters_for_export: dict[str, dict[str, Any]] = {
+            "image_encoder": {
+                "input_size": (1, 3, self.model.image_size, self.model.image_size),
+                "mean": (123.675, 116.28, 103.53),
+                "std": (58.395, 57.12, 57.375),
+                "resize_mode": "fit_to_window",
+            },
+            "decoder": {
+                "input_size": (
+                    1,
+                    self.model.embed_dim,
+                    self.model.image_embedding_size,
+                    self.model.image_embedding_size,
+                ),
+            },
+        }
+
+    def export(  # type: ignore[override]
+        self,
+        output_dir: Path,
+        base_name: str,
+        export_format: OTXExportFormatType,
+        precision: OTXPrecisionType = OTXPrecisionType.FP32,
+    ) -> dict[str, Path]:
+        """Export the model to the specified format."""
+        model = {
+            "image_encoder": self.model.image_encoder,
+            "decoder": self.model,
+        }
+        dummy_inputs = {
+            "image_encoder": {
+                "images": torch.randn(1, 3, self.model.image_size, self.model.image_size, dtype=torch.float32),
+            },
+            "decoder": {
+                "image_embeddings": torch.zeros(
+                    1,
+                    self.model.embed_dim,
+                    self.model.image_embedding_size,
+                    self.model.image_embedding_size,
+                    dtype=torch.float32,
+                ),
+                "point_coords": torch.randint(low=0, high=1024, size=(1, 2, 2), dtype=torch.float32),
+                "point_labels": torch.randint(low=0, high=4, size=(1, 2), dtype=torch.float32),
+                "mask_input": torch.randn(
+                    1,
+                    1,
+                    4 * self.model.image_embedding_size,
+                    4 * self.model.image_embedding_size,
+                    dtype=torch.float32,
+                ),
+                "has_mask_input": torch.tensor([[1]], dtype=torch.float32),
+                "orig_size": torch.randint(low=256, high=2048, size=(1, 2), dtype=torch.int64),
+            },
+        }
+        output_names = {
+            "image_encoder": ["image_embeddings"],
+            "decoder": ["upscaled_masks", "iou_predictions", "low_res_masks"],
+        }
+        dynamic_axes = {
+            "image_encoder": None,
+            "decoder": {
+                "point_coords": {1: "num_points"},
+                "point_labels": {1: "num_points"},
+            },
+        }
+
+        export_paths: dict[str, Path] = {}
+        for module in ["image_encoder", "decoder"]:
+            self._export_parameters = module  # type: ignore[assignment]
+            export_paths[module] = self._exporter.export(
+                model=model[module],
+                output_dir=output_dir,
+                base_model_name=f"visual_prompting_{module}",
```

Comment:
```suggestion
                base_model_name=f"{base_name}_{module}",
```
It's a minor one but how about changing like this?

Reply:
Do you think `exported_model_{module}` seems better than `visual_prompting_{module}`?

Reply:
I like both but my intention is for keeping unity with other tasks and not ignoring `base_name` argument.

Reply:
I agree with that, updated at https://github.com/openvinotoolkit/training_extensions/pull/2955/commits/84ed9f4f1769453e21f59015110eee3002168d64.